### DISCUSSION TITLE
What happens to EguiContext.

### DISCUSSION THREAD

**Discussion by MiniMinerX** [rewin123/space_editor#313] [Q&A]
I have this function fn update_worldspace(
    mut ctxs: Query<&mut bevy_egui::EguiContext, With<EguiRenderToTextureHandle>>,
    time: Res<Time>,
    mut commands: Commands,
    mut curves: Query<(Entity, &mut CurveContainer, &mut EditorWindow, &mut AnimateWithCurve)>,
) {

    for mut ctx in ctxs.iter_mut() {
        egui::Window::new("other ui").show(ctx.get_mut(), |ui| {
            curve_ui(
                ui,
                &time,
                &mut commands,
                &mut curves
            );
        });
    }
} My entities with the RenderToTexture are supposed to have an EguiContext with them. Without the editor, this registers fine but with the editor, it doesnt and I dont know why.

**Comment by naomijub** [comment]
Not exactly sure why, but I do think that the editor context is only available in editor mode. If you need egui context on your game we might need to do some changes. For now, I suggest have a new context for your game mode.

Another alternative is if you share you code with one of us. Would be probably easier to identify the specific issue

**Comment by rewin123** [comment]
Do you have a screenshot of how the error happens, or an error log? Entities stop showing up in the texture where they were supposed to be displayed?
